parachain/pallets/asset/src/lib.rs

```diff
@@ -39,9 +39,9 @@ decl_event!(
 	where
 		AccountId = <T as system::Trait>::AccountId,
 	{
-		Burned(AssetId, AccountId, U256),
-		Minted(AssetId, AccountId, U256),
-		Transferred(AssetId, AccountId, AccountId, U256),
+		Burned(H160, AccountId, U256),
+		Minted(H160, AccountId, U256),
+		Transferred(H160, AccountId, AccountId, U256),
```

Comment:
Yeah, I had to make this change purely for UX purposes. The problem is that the Polkadot JS app already has a non-overridable type mapping for the `AssetId` parameter. The similarly named `assets` pallet in FRAME also emits an event which includes a parameter named `AssetId`, and Polkadot-JS It interpets it as an unsigned integer rather than the 20-byte hash we need.

What we really need to do here is find an alias for `H160` that is semantically correct. maybe `EthereumAddr` or `TokenID`. I dunno.

Reply:
Added ticket to track: #81